### PR TITLE
Add experimental OPs.

### PR DESCRIPTION
**Pull request by ebarsoum**
Those are experimental OPs not ready to be put in core ONNX.

### CODE REVIEW

File: onnx/defs/experiments/defs.cc [hunk 1]
```diff
@@ -363,3 +363,87 @@ OPERATOR_SCHEMA(ATen)
...
+    .Attr("bias", "Bias applied to each channel, same size as C.", AttrType::FLOATS)
+    .Attr("scale", "(float, default 1.0) the scale to apply.", AttrType::FLOAT)
+    .Input(0, "input", "Input tensor of shape [N,C,H,W]", "T")
+    .Output(0, "output", "Result, has same shape and type as input", "T")
+    .TypeConstraint(
+        "T",
+        {"tensor(float16)", "tensor(float)", "tensor(double)"},
+        "Constrain input and output types to float tensors.");
+
+OPERATOR_SCHEMA(MeanSubtraction)
```
Comment (bddppq) [inline onnx/defs/experiments/defs.cc]: Is this covered by Sub (by putting the attribute "image" as the second input)?

Comment (ebarsoum) [reply onnx/defs/experiments/defs.cc]: Agree removed.

**Comment by ebarsoum** [comment]
@bddppq  I renamed and updated Normalzie to LpNormalization to be consistent and more flexible.

File: onnx/defs/experiments/defs.cc [hunk 2]
```diff
@@ -299,8 +299,10 @@ OPERATOR_SCHEMA(Normalize)
         {"tensor(float16)", "tensor(float)", "tensor(double)"},
         "Constrain input and output types to float tensors.")
     .SetDoc(R"DOC(
-Given a matrix, apply L2-normalization along the last dimension.
-)DOC");
+Given a matrix, apply Lp-normalization along the provided axis.
+)DOC")
+    .Attr("axis", "(int64, default -1) the axis on which to apply normalization, -1 mean last axis.", AttrType::INT)
+    .Attr("p", "(float, default 2.0) the order of the normalization, only 2.0 is supported.", AttrType::FLOAT);
```
Comment (bddppq) [inline onnx/defs/experiments/defs.cc]: If only 2.0 is supported, how about renaming the op to L2Normalization and remove this attribute? :-)

File: onnx/defs/experiments/defs.cc [hunk 3]
```diff
@@ -363,3 +365,73 @@ OPERATOR_SCHEMA(ATen)
 Experimental allowing ATen operations to be accessed directly from Caffe2
 to allow for quick prototyping when ONNX is missing standard versions of
 and op)DOC");
+
+
+OPERATOR_SCHEMA(ImageScaler)
```
Comment (bddppq) [inline onnx/defs/experiments/defs.cc]: Is it equivalent to Scale followed by Add?

File: onnx/defs/experiments/defs.cc (hunk 2 above)
Comment (ebarsoum) [reply onnx/defs/experiments/defs.cc]: I am planning to move it from experiment to core, and I want to make it generic so in the future we can add p=1.0.

File: onnx/defs/experiments/defs.cc (hunk 3 above)
Comment (ebarsoum) [reply onnx/defs/experiments/defs.cc]: Yes, but we have one fused OP that does both at once. I put it in experiment with the attention to remove it in the future.

**Comment by Yangqing** [comment]
Travis seems to have passed but it's not updated back to github yet: https://travis-ci.org/onnx/onnx/builds/299981704?utm_source=github_status&utm_medium=notification consider it already passing.

**Pull request merged**